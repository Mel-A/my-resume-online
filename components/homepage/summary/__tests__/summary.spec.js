import { shallow } from 'enzyme'
import Summary from '..'

test('summary renders correctly', () => {
  const summary = shallow(<Summary />)
  expect(summary).toMatchSnapshot()
});