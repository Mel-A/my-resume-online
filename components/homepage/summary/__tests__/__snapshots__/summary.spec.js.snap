// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`summary renders correctly 1`] = `ShallowWrapper {}`;
